sample.mvn.parent/pom.xml

```diff
@@ -1,7 +1,9 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+>
 	<modelVersion>4.0.0</modelVersion>
+
 	<groupId>sample.mvn</groupId>
 	<artifactId>sample.mvn.parent</artifactId>
 	<version>0.0.1-SNAPSHOT</version>
```

Comment:
I tried to used the `${revision}` property in the parent pom so children will inherit that one too. I also tried defining the default value (0.0.1-SNAPSHOT) inside the .mvn folder, but the project fails to run from the command line, even if I pass the property as part of the command with` -Drevision=0.0.1-SNAPSHOT`>
I think this is due to the app being launched from the launch module and somehow the other children modules do not get the `${revison}` variable resolved.
But most likely I am doing this wrong. Is there a way to solve that?

Reply:
Sorry no idea